### DISCUSSION TITLE
Remove popup close button

### DISCUSSION THREAD

**Discussion by pgale** [Clooos/Bubble-Card#525] [General]
I use a series of popups on my Lenovo ThinkSmart View panels. They work pretty well. I do find that the popup close button gets in the way and uses valuable screen real estate though. I'm happy using the horizontal stack buttons to open/close the popups so is it possible through CSS etc to remove the close button completely and shift up the content?

If not, this would need to be a feature request. 

It would seem to be a logical feature. 

Thanks.

**Comment by Clooos** [comment]
This is now possible in v2.1.0-beta.2 but I guess that you've seen it👌

**Reply by pgale** [reply]
Yes thanks! I've yet to try it as I'm still using the CSS- will try it today. Thanks.

**Reply by ncodee** [reply]
> This is now possible in v2.1.0-beta.2 but I guess that you've seen it👌

Would it possible to add an option to remove the close button, without hiding the whole header section? thanks


```
.bubble-close-button {
  display: none !important;
}
```

I know you can use the css code above, but it would be nice to have a toggle option, with a nice little tab indicator on top of the bottom sheet as well.  Something [like this](https://legion.telkom.design/components/bottomsheet/guideline).

Thanks
